crates/db_schema/src/source/local_user.rs

```diff
@@ -69,38 +68,59 @@ pub struct LocalUser {
   pub collapse_bot_comments: bool,
 }
 
-#[derive(Clone, TypedBuilder)]
-#[builder(field_defaults(default))]
+#[derive(Clone, derive_new::new)]
```

Comment:
I got rid of the typed builder, and used the same derive_new technique that's used for the `PersonInsertForm`